tests/endpoint-test.js

```diff
@@ -1152,6 +1172,35 @@ describe('Endpoint', () => {
 						done();
 					});
 			});
+			it('should extend the retention policy expiration date for the file', done => {
+				const fileID = '1234567890';
+				const updates = {
+					disposition_at: '2023-12-12T10:53:43-08:00',
+				};
+				const fixture = getFixture('files/put_files_disposition_at_custom_params_200');
+				apiMock
+					.put(`/2.0/files/${fileID}?fields=disposition_at`, updates)
```

Comment:
I do not think API supports PUT with query params?
https://developer.box.com/reference/put-files-id/

Reply:
Hi @antusus 

It's have `fields` query params here: https://developer.box.com/reference/put-files-id/#param-file_id